DuckDuckGo/AppDependencyProvider.swift

```diff
@@ -149,11 +149,15 @@ class AppDependencyProvider: DependencyProvider {
             }
             return { nil }
         }()
+#if os(macOS)
         networkProtectionKeychainTokenStore = NetworkProtectionKeychainTokenStore(keychainType: .dataProtection(.unspecified),
                                                                                   serviceName: "\(Bundle.main.bundleIdentifier!).authToken",
                                                                                   errorEvents: .networkProtectionAppDebugEvents,
                                                                                   isSubscriptionEnabled: accountManager.isUserAuthenticated,
                                                                                   accessTokenProvider: accessTokenProvider)
+#else
+        networkProtectionKeychainTokenStore = NetworkProtectionKeychainTokenStore(accessTokenProvider: accessTokenProvider)
```

Comment:
Ideally we could remove this entirely from iOS, though I understand we still have some code at the BSK layer that expects it.